src/hooks/useSelectedRevisions.ts

```diff
@@ -72,8 +76,20 @@ const useSelectRevision = () => {
 
   const deleteSelectedRevisions = (revision: RevisionsList) => {
     const newSelected = [...selectedRevisions];
```

Comment:
nit: this clone can happen after the if block: we need it for the `splice` operation, but not the `slice` operation (slice will always return a new array, while `splice` modifies the array).